src/js/components/AddClient/index.js

```diff
@@ -1,79 +1,96 @@
+      inputList: inputList.concat(<RecipientInput />)
+    })
+  }
+
+  render () {
     return (
-      <div className="modal-container" style={{height: 200}}>
+      <div className='modal-container' style={{height: 200}}>
         <Button
-          bsStyle="primary"
-          bsSize="large"
+          bsStyle='primary'
+          bsSize='large'
           onClick={this.props.toggleModal}
         >
-          Add a Patient
+          Add New Patient Letters
         </Button>
 
         <Modal
           show={this.props.showModal}
           onHide={this.props.toggleModal}
+          bsSize={'lg'}
         >
           <Modal.Header closeButton>
-            <Modal.Title id="contained-modal-title">Client Form</Modal.Title>
+            <Modal.Title id='contained-modal-title'>Client Form</Modal.Title>
           </Modal.Header>
           <Modal.Body>
             <Form horizontal>
-              <FormGroup controlId="formHorizontalEmail">
-                <Col componentClass={ControlLabel} sm={2}>
-                  First Name
-                </Col>
-                <Col sm={10}>
-                  <FormControl type="email" placeholder="Email" />
-                </Col>
-              </FormGroup>
-              <FormGroup controlId="formHorizontalEmail">
+              <FormGroup controlId='formHorizontalEmail'>
                 <Col componentClass={ControlLabel} sm={2}>
-                  Last Name
+                  Patient ID
                 </Col>
                 <Col sm={10}>
-                  <FormControl type="email" placeholder="Email" />
+                  <FormControl type='email' placeholder='Patient ID' />
                 </Col>
               </FormGroup>
-              <FormGroup controlId="formHorizontalEmail">
+              <FormGroup controlId='formHorizontalEmail'>
                 <Col componentClass={ControlLabel} sm={2}>
                   Email
                 </Col>
                 <Col sm={10}>
-                  <FormControl type="email" placeholder="Email" />
+                  <FormControl type='email' placeholder='Email' />
                 </Col>
               </FormGroup>
-              <FormGroup controlId="formHorizontalEmail">
+              <FormGroup controlId='formHorizontalEmail'>
                 <Col componentClass={ControlLabel} sm={2}>
-                  Phone Number
+                  Mobile Number
                 </Col>
                 <Col sm={10}>
-                  <FormControl type="number" placeholder="Email" />
+                  <FormControl type='email' placeholder='Mobile Number' />
                 </Col>
               </FormGroup>
+              <RecipientInput />
+              <Col smOffset={9} sm={3}>
+                <Button bsStyle='primary' onClick={this.handleClick}>
+                  Add New Recipient
+                </Button>
+              </Col>
+              {this.state.inputList.map((input, index) => {
+                return input
+              })}
```

Comment:
About lines 73-75, I think the `.map` is superfluous - I think you could just put `{this.state.inputList}` instead.